contextproject-ms3/src/main/java/contextproject/App.java

```diff
@@ -1,23 +1,26 @@
 package contextproject;
 
-import contextproject.controllers.CLIController;
-import contextproject.controllers.WindowController;
-
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-
 import javafx.application.Application;
 import javafx.fxml.FXMLLoader;
 import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import contextproject.controllers.CLIController;
+import contextproject.controllers.WindowController;
+import contextproject.loaders.FolderLoader;
+import contextproject.models.Playlist;
+
 /**
  * Hello world.
  *
  */
 public class App extends Application {
   static Logger log = LogManager.getLogger(App.class.getName());
+  static final String DIRECTORY = "/Users/daveystruijk/Documents/FEESJE/House (Chill)";
```

Comment:
Use File.separator and user.dir to get the users directory. This wil only work on your machine